tests/dependency_analyzer/expected_results/apache_maven_with_sbom_provided.dl

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved. */
+/* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
+
+#include "prelude.dl"
+
+Policy("check-dependencies", component_id, "Check the dependencies of pkg:github.com/timyarkov/multibuild_test@a8b0efe24298bc81f63217aaa84776c3d48976c5") :-
+    transitive_dependency(component_id, dep_0),
+    is_component(dep_0, "pkg:maven/org.junit.jupiter/junit-jupiter-engine@5.8.1?type=jar"),transitive_dependency(component_id, dep_1),
+    is_component(dep_1, "pkg:maven/org.hamcrest/hamcrest-core@2.2?type=jar").
```

Comment:
I presume this corresponds to `apache_maven_with_sbom_provided.json`? I see that the original also checks for a bunch of other dependencies with `"available": "MISSING REPO URL"` that are not included in the policy version. What is the reason for this?

Also, this policy checks for the presence of the expected dependencies, but does not forbid additional unexpected dependencies (whereas the original test did).

Reply:
Also, it would be more readable if it were not formatted with the `transitive_dependency` for the next component on the same line as the last one.

Reply:
Looks like dependency related policies will not be included and we will continue to use JSON comparisons for those tests.